### PR TITLE
OTJ 13 cards

### PR DESCRIPTION
**Pull request by Simisays**

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/freestrider_commando.txt [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Name:Freestrider Commando
+ManaCost:2 G
+Types:Creature Centaur Mercenary
+PT:3/3
+K:etbCounter:P1P1:2:CheckSVar$ ETB:CARDNAME enters the battlefield with two +1/+1 counters on it if it wasn't cast or no mana was spent to cast it.
+SVar:ETB:Count$Valid Card.Self+wasNotCast,Card.Self+CastSa Spell.ManaSpent EQ0
```
Comment (Simisays) [inline forge-gui/res/cardsfolder/upcoming/freestrider_commando.txt:6]: This isn't working

File: forge-gui/res/cardsfolder/upcoming/great_train_heist.txt [hunk 2]
```diff
@@ -0,0 +1,13 @@
+Name:Great Train Heist
+ManaCost:R
+Types:Instant
+K:Spree 
+A:SP$ Charm | Choices$ DBUntapAll,DBPumpAll,DBCombatTrig | MinCharmNum$ 1 | CharmNum$ 3 | Spree$ True
+SVar:DBUntapAll:DB$ UntapAll | SpreeCost$ 2 R | ValidCards$ Creature.YouCtrl | SubAbility$ DBAddCombat | SpellDescription$ Untap all creatures you control. If it's your combat phase, there is an additional combat phase after this phase.
+SVar:DBAddCombat:DB$ AddPhase | ConditionPhases$ COMBAT_BEGIN,COMBAT_DECLARE_ATTACKERS,COMBAT_DECLARE_BLOCKERS,COMBAT_FIRST_STRIKE_DAMAGE,COMBAT_DAMAGE,COMBAT_END | AfterPhase$ EndCombat | ExtraPhase$ Combat
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/great_train_heist.txt:7]: `ConditionPhases$ BeginCombat->EndCombat`

File: forge-gui/res/cardsfolder/upcoming/holy_cow.txt [hunk 3]
```diff
@@ -0,0 +1,11 @@
+Name:Holy Cow
+ManaCost:2 W
+Types:Creature Ox Angel
+PT:2/2
+K:Flash
+K:Flying
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigGainLife | TriggerDescription$ When CARDNAME enters the battlefield, you gain 2 life and scry 1. (Look at the top card of your library. You may put that card on the bottom.)
+SVar:TrigGainLife:DB$ GainLife | LifeAmount$ 2 | SubAbility$ DBScry
+SVar:DBScry:DB$ Scry | ScryNum$ 2
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/holy_cow.txt:9]: 1

File: forge-gui/res/cardsfolder/upcoming/mobile_homestead.txt [hunk 4]
```diff
@@ -0,0 +1,11 @@
+Name:Mobile Homestead
+ManaCost:2
+Types:Artifact Vehicle
+PT:3/3
+S:Mode$ Continuous | Affected$ Card.Self | AddKeyword$ Haste | IsPresent$ Mount.YouCtrl | Description$ CARDNAME has haste as long as you control a Mount.
+T:Mode$ Attacks | ValidCard$ Card.Self | Execute$ DBDig | TriggerDescription$ Whenever CARDNAME attacks, look at the top card of your library. If it's a land card, you may put it onto the battlefield tapped.
+SVar:DBDig:DB$ Dig | DigNum$ 1 | ChangeNum$ 1 | ChangeValid$ Card.Land | ForceRevealToController$ True | Optional$ True | DestinationZone$ Battlefield | DestinationZone2$ Library | LibraryPosition2$ 0
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/mobile_homestead.txt:7]: tapped

File: forge-gui/res/cardsfolder/upcoming/freestrider_commando.txt (hunk 1 above)
Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/freestrider_commando.txt:6]: imo just Compare against `Count$CastTotalManaSpent`?

File: forge-gui/res/cardsfolder/upcoming/miriam_herd_whisperer.txt [hunk 5]
```diff
@@ -0,0 +1,10 @@
+Name:Miriam, Herd Whisperer
+ManaCost:G W
+Types:Legendary Creature Human Druid
+PT:3/2
+S:Mode$ Continuous | Affected$ Mount.YouCtrl,Vehicle.YouCtrl | AddKeyword$ Hexproof | Condition$ PlayerTurn | Description$ As long as it's your turn, Mounts and Vehicles you control have hexproof.
+T:Mode$ Attacks | ValidCard$ Mount.YouCtrl,Vehicle.YouCtrl | Execute$ TrigPutCounter | TriggerZones$ Battlefield | TriggerDescription$ Whenever a Mount or Vehicle you control attacks, put a +1/+1 counter on it.
+SVar:TrigPutCounter:DB$ PutCounter | Defined$ TriggeredAttackerLKICopy | CounterType$ P1P1 | CounterNum$ 1
+DeckHas:Ability$Counters
+DeckHints:Type$Mount|Vehicle
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/miriam_herd_whisperer.txt:9]: I think Needs – this is bad without those types

File: forge-gui/res/cardsfolder/upcoming/spring_splasher.txt [hunk 6]
```diff
@@ -0,0 +1,7 @@
+Name:Spring Splasher
+ManaCost:1 U
+Types:Creature Frog Beast
+PT:2/1
+T:Mode$ Attacks | ValidCard$ Card.Self | Execute$ TrigPump | TriggerDescription$ Whenever CARDNAME attacks, target creature defending player controls gets -3/-0 until end of turn.
+SVar:TrigPump:DB$ Pump | ValidTgts$ Creature.ControlledBy TriggeredDefendingPlayer | TgtPrompt$ Select target creature defending player controls | NumAtt$ -3 | IsCurse$ True
+Oracle:Whenever Spring Splasher attacks, target creature defending player controls gets -3/-0 until end of turn.
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/spring_splasher.txt:7]: should this have the HasAttackEffect:TRUE line?

File: forge-gui/res/cardsfolder/upcoming/great_train_heist.txt [hunk 7]
```diff
@@ -0,0 +1,13 @@
+Name:Great Train Heist
+ManaCost:R
+Types:Instant
+K:Spree 
+A:SP$ Charm | Choices$ DBUntapAll,DBPumpAll,DBCombatTrig | MinCharmNum$ 1 | CharmNum$ 3 | Spree$ True
+SVar:DBUntapAll:DB$ UntapAll | SpreeCost$ 2 R | ValidCards$ Creature.YouCtrl | SubAbility$ DBAddCombat | SpellDescription$ Untap all creatures you control. If it's your combat phase, there is an additional combat phase after this phase.
+SVar:DBAddCombat:DB$ AddPhase | ConditionPhases$ BeginCombat->EndCombat | AfterPhase$ EndCombat | ExtraPhase$ Combat
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/great_train_heist.txt:7]: ConditionPlayerTurn missing

**Review by tool4ever — APPROVED** [review]

**Pull request merged**